packages/storybook/stories/va-text-input-uswds.stories.jsx

```diff
@@ -60,7 +65,9 @@ const defaultArgs = {
   'charcount': false,
   'max': undefined,
   'min': undefined,
-  'currency': undefined
+  'currency': undefined,
```

Comment:
Since we will have prefix with icons supported, the old `currency` prop seems redundant now. I wonder if it would be good to create a follow-up ticket to deprecate this. 🤔

Reply:
Yes, I had that in mind. I can create a follow-up ticket to deprecate the `currency` prop.

Reply:
I created a new ticket.

Reply:
@harshil1793 Thanks Harshil! Can you link to that ticket?